js/start.js

```diff
@@ -0,0 +1,11 @@
+import $ from 'jquery';
+import app from './app';
+import LocalSettings from './models/LocalSettings';
+import PageNav from './views/PageNav.js';
+
+app.localSettings = new LocalSettings({ id: 1 });
+app.localSettings.fetch().fail(() => app.localSettings.save());
```

Comment:
The fetch will fail the first time you load the app because the model was never saved in local storage. In that case, the save will save whatever the default options are.